HW/10.27/Web_10.27DZ/Web_10.27DZ/PersonEnv/PeopleStorage.cs

```diff
@@ -69,9 +70,9 @@ public void UpDate(Guid id, Person item)
             }
         }
 
-        public void WriteDataInFile(IJsonIteractor _jsonIteractor, IConfiguration _configuration)
+        public void Save(DI dI)
         {
-            _jsonIteractor.JsonWriteList(_configuration, InnerCol);
+            dI.jsonIteractor.JsonWriteList(dI.configuration, InnerCol);
```

Comment:
why do you use this `DI dI`?

just pass services that you need from DI to ctor and store them in fields like in controller